internal/evaluator/conftest_evaluator.go

```diff
@@ -31,11 +32,38 @@ import (
 	"github.com/hacbs-contract/ec-cli/internal/utils"
 )
 
+type contextKey string
+
+const clientContextKey contextKey = "ec.evaluator.client"
+
+type client interface {
```

Comment:
Perhaps something like this?

```suggestion
type testRunner interface {
```

Perhaps for interfaces with a single function we don't need the interface but could do something like:

```suggestion
type testRunner func(context.Context, []string) ([]output.CheckResult, error)
```